pkg/render/intrusiondetection/dpi/dpi_test.go

```diff
@@ -243,6 +268,47 @@ var _ = Describe("DPI rendering tests", func() {
 		validateDPIComponents(resources, false)
 	})
 
+	It("should render all resources for deep packet inspection with default resource requirements for a management cluster without DPI resources", func() {
```

Comment:
this test name is a bit much - does the test care at all about resource requirements? 

let's focus it in on just the thing we want to verify here:

"Should render Linseed permissions for management clusters, even without DPI enabled"